lona/html/parsing.py

```diff
@@ -161,7 +161,11 @@ def handle_data(self, data):
             if not text:
                 return
 
-        self._node.append(TextNode(text))
+        if self._node.tag_name == 'textarea':
+            self._node.value = text
```

Comment:
I'm afraid it is complicated than that. For example following should pass.
```py
assert HTML('<textarea>abc<br>xyz</textarea>')[0].value == 'abc<br>xyz'
assert HTML('<textarea>aaa <b>bbb</b> ccc</textarea>')[0].value == 'aaa <b>bbb</b> ccc'
```

Reply:
@maratori You are right. That's what the javascript does too

Reply:
@maratori Fixed!